packages/main/types/requests.ts

```diff
@@ -114,3 +117,136 @@ export interface RequestOptions {
    */
   apiVersion?: string;
 }
+/**
+ * Defines a tool that model can call to access external knowledge.
+ * @public
+ */
+export declare type Tool = FunctionDeclarationsTool;
+
+/**
+ * Structured representation of a function declaration as defined by the
+ * [OpenAPI 3.0 specification](https://spec.openapis.org/oas/v3.0.3). Included
+ * in this declaration are the function name and parameters. This
+ * FunctionDeclaration is a representation of a block of code that can be used
+ * as a Tool by the model and executed by the client.
+ * @public
+ */
+export declare interface FunctionDeclaration {
```

Comment:
Can we lift this to the level of `Tool` as described in https://docs.google.com/document/d/1Z0xNfOoUs8sIwSpJjAPNBvL5jVmM-kDJ1MmElfCk2Wc/edit?tab=t.0#heading=h.x9snb54sjlu9 ?

Reply:
Note: we decided not to do this, we'll leave it as is.